### ISSUE TITLE
SNOW-251526: Cannot Build on Windows

### ISSUE THREAD

**Issue by daffyduck25** [snowflakedb/pdo_snowflake#236]
Windows 10
x64
PHP V 7.3.0
VS2017

Hello.  I am trying to build the .dll from instructions here: https://github.com/snowflakedb/pdo_snowflake#building-the-driver-on-windows on a Windows machine. 

I bypassed the php build instructions as I already have PHP on my machine.

When I get to this section:

.\scripts\run_build_pdo_snowflake.bat <arch> <build> <visual studio version> <full PHP version> <path to PHP SDK>

After the files are copied I receive an error the 'buildconf' is not recognizable.

Did I incorrectly assume that I could skip the php build section?  Or is there something else I should  be doing?

Thank you.

![image](https://user-images.githubusercontent.com/59655689/103037440-d7265c80-4539-11eb-9225-f781eab46c27.png)

**Comment by Keith-Wade** [comment]
We had same issues tryin to compile for PHP 7.4.15 and ran into the buildconf error.
We got around that by copying the PHP 7.4.15 source into the PHP SDK library that was built in a previous step:
**_C:\php-sdk\phpmaster\vc15\x64\php-src_**

This solved our problem with building the php_pdo_snowflake.dll.  However, we couldn't load it into our PHP instance.
We were able to get the dll loaded into the PHP 7.4.15 instance that was compiled during the snowflake dll build.  We're thinking there must be some PHP build issues between the Snowflake PHP build and the pre-built dlls from the PHP site we're using.
